bugsnag/client.py

```diff
@@ -150,6 +165,8 @@ class ClientContext(object):
     def __init__(self, client, exception_types=None, **options):
         self.client = client
         self.options = options
+        if 'severity' in options:
+            options['severity_reason'] = dict(type='contextSpecifiedSeverity')
```

Comment:
Should this be `userContextSetSeverity`?

Reply:
This one seems to already have been fixed